index.bs

```diff
@@ -953,17 +953,9 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 							1. If |type| is not in the [=mandatory data types=] list, then [=reject=] |p| with {{"NotAllowedError"}} {{DOMException}} in |realm| and abort these steps.
 
-							1. Let |cleanItem| be a sanitized copy of |blob|.
-
-								Issue: Add definition of sanitized copy.
-
-							1. If unable to create a sanitized copy, then follow the below steps:
```

Comment:
Maybe change to be:

1. If sanitization was attempted and was not successfully completed, then follow the below steps:

Reply:
Done.